config/environments/development.rb

```diff
@@ -75,6 +75,9 @@
   config.action_mailer.perform_caching = false
 
   config.hosts << ENV["APP_DOMAIN"] unless ENV["APP_DOMAIN"].nil?
+  if (gitpod_workspace_url = ENV["GITPOD_WORKSPACE_URL"])
```

Comment:
I'm not super happy about having this in` development.rb`, but I didn't feel like doing anything too clever given that Github Codespaces could conceivably replace Gitpod in the not too distant future.